studio/lib/interfaces/global.ts

```diff
@@ -8,9 +8,26 @@ export interface DocumentWithSlug {
   _updatedAt: string;
 }
 
-export type InternationalizedString = InternationalizedStringValue[];
+export function isInternationalizedString(
```

Comment:
this is a function but placed inside the interfaces folder?

Reply:
This is a type guard for the type defined in this file. So I valued closeness to the relevant type over having only types/interfaces in the file.